src/style.css

```diff
@@ -0,0 +1,25 @@
+html {
+  margin: 32px 22px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-size: auto;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--1 > div:nth-child(1),
+.stars--2 > div:nth-child(-n + 2),
+.stars--3 > div:nth-child(-n + 3),
+.stars--4 > div:nth-child(-n + 4),
+.stars--5 > * {
```

Comment:
don't use *. What if we need to add some elements inside the block(not stars__star)